### PR TITLE
Integrate Extension Explorer

### LINKED ISSUE #743: Declare the versions of core extensions in use
On a call with @duncandewhurst and @timgdavies, we discussed the desire, in future releases of OCDS, to declare the version numbers of core extensions in use within each release of OCDS.

A new release of a core extension isn't necessarily made with each new release of OCDS (e.g. if OCDS changes, but the extension doesn't). Looking ahead, we might also want to make releases of core extensions at a different rhythm than releases of OCDS.

This would mean that the version number of a core extension may not match the version number of OCDS. As such, we should provide a way for users of the standard documentation to determine which version number of core extensions to use with a given release of OCDS.

This might take the form of a table on an appropriate page within the documentation.

### LINKED ISSUE #803: Link to Extension Explorer instead of building extension docs
Update the documentation to refer to specific versions of extensions in the extensions explorer, instead of including extension documentation and extension lists.

Will involve changes to the scripts that pull in the extension docs.

### PR DESCRIPTION
**Pull request by jpmckinney**
Fixes #803

* Merge extensions/party_details into schema/reference and extensions/developing into extensions. Use extensionexplorerlinklist on extensions page ([preview](http://standard.open-contracting.org/extensionexplorer/en/extensions/#using-extensions)).
* Remove community extensions page, page translation, toctree item, .tx/config entry, test. Remove extensionselectortable tables and related JavaScript.
* Remove core extension pages, page translations, codelists, toctree items, .tx/config entries. Update core extensions URLs in changelog.
* Update references to bids and location extensions on schema reference page.

Also, fixes #743

### CODE REVIEW

**Comment by jpmckinney** [comment]
@duncandewhurst This PR tries to make minimal changes in order to remove the individual extension pages. (There are many commits starting from "Report lang in error in test_docs.py" to try to find where the last references to extensions pages were.)

You can focus on the Markdown pages for the review, the biggest change being to `extensions/index.md` https://github.com/open-contracting/standard/pull/822/files#diff-84ac45d4c129eb5f31f27d64f9b33ab1

File: standard/docs/en/extensions/index.md [hunk 1]
```diff
@@ -14,66 +14,23 @@ All core and community extensions are listed in the [extensions registry](https:
...
 
-```eval_rst
-.. extensionselectortable::
-   :group: community
 ```
 
+## Developing your own extensions
 
-## More information
+If you have additional fields to include in your data which are not covered by an existing extension, you may create a new extension.
```
Comment (duncandewhurst) [inline standard/docs/en/extensions/index.md]: I suggest making the meaning of 'covered by' explicit, making it explicit that additional fields should be included in OCDS data and changing the recommendation to create a new extension from 'may' to 'should'. 

> If you have additional fields which cannot be mapped to the OCDS schema or an existing extension, you should include these in your OCDS data and create a new extension to document their structure and meaning.

File: standard/docs/en/extensions/index.md [hunk 2]
```diff
@@ -14,66 +14,23 @@ All core and community extensions are listed in the [extensions registry](https:
...
-}
---></style> 
-
-```json
-{
-    "extensions":[],
-    "releases":[]
-}
-```
+Extensions are applied by adding their URLs to the `extensions` array in the release package or record package. You can discover extensions and read their documentation using the [Extension Explorer](https://extensions.open-contracting.org/en/).
```
Comment (duncandewhurst) [inline standard/docs/en/extensions/index.md]: It's not clear from this page what an extension's URL is. We should either include a static example of how an extension is declared, explain that it is the URL of the `extension.json` file or update the text to make it clear the URLs can be found from the extension explorer:

> Extensions are applied by adding their URLs to the `extensions` array in the release package or record package. You can discover extensions, read their documentation and find the URL to add using the [Extension Explorer](https://extensions.open-contracting.org/en/).

**Comment by duncandewhurst** [comment]
The ``extensionList`` directive displays core extensions twice, under both core and community headings:

image

File: standard/docs/en/schema/reference.md [hunk 3]
```diff
@@ -480,7 +482,7 @@ As well as providing this machine-readable link between processes, publishers ma
 
 ### Location
 
-The addition of location information is currently handled through a [proposed extension](../extensions/location.md) to the standard.
+The optional [Location](http://extensions.open-contracting.org/en/extensions/location/) extension can be used to provide location information.
```
Comment (duncandewhurst) [inline standard/docs/en/schema/reference.md]: Should this be a link to the correct version of the extension for this OCDS release? (the current link redirects to the master branch)

File: standard/docs/en/schema/reference.md [hunk 4]
```diff
@@ -141,10 +146,7 @@ It may contain details of a forthcoming process to receive and evaluate proposal
 
 ### Bids
 
-```eval_rst
-.. extensionlist:: The optional bids extension can be used to provide summary and detailed information about bids.
-   :list: bids
-```
+The optional [Bid statistics and details](http://extensions.open-contracting.org/en/extensions/bids/) extension can be used to provide bid statistics and detailed information about individual bids.
```
Comment (duncandewhurst) [inline standard/docs/en/schema/reference.md]: Should this be a link to the correct version of the extension for this OCDS release? (the current link redirects to the master branch)

File: standard/docs/en/extensions/index.md [hunk 5]
```diff
@@ -14,66 +14,23 @@ All core and community extensions are listed in the [extensions registry](https:
...
 
-## Core extensions
+This version of OCDS uses these specific versions of core extensions:
```
Comment (duncandewhurst) [inline standard/docs/en/extensions/index.md:19]: This page still uses the terminology of 'core' extensions but the extensions explorer uses 'recommended'

**Review by duncandewhurst — COMMENTED** [review]
@jpmckinney I've reviewed and left individual comments on areas where changes might be required.

**Comment by jpmckinney** [comment]
> The extensionList directive displays core extensions twice, under both core and community headings:

This should be fixed once the last commit is built: http://standard.open-contracting.org/extensionexplorer/en/schema/reference/#milestone 

**Update:** `extensions.js` seems to get cached, so you might need to open a private window to test.

File: standard/docs/en/extensions/index.md (hunk 5 above)
Comment (jpmckinney) [reply standard/docs/en/extensions/index.md:19]: Changing that language in the OCDS docs would be something for a later release. I created #833.

File: standard/docs/en/schema/reference.md (hunk 4 above)
Comment (jpmckinney) [reply standard/docs/en/schema/reference.md]: Done in 127b5d4

File: standard/docs/en/schema/reference.md (hunk 3 above)
Comment (jpmckinney) [reply standard/docs/en/schema/reference.md]: Done in 127b5d4

File: standard/docs/en/extensions/index.md (hunk 1 above)
Comment (jpmckinney) [reply standard/docs/en/extensions/index.md]: Done in f3cd65c

File: standard/docs/en/extensions/index.md (hunk 2 above)
Comment (jpmckinney) [reply standard/docs/en/extensions/index.md]: For now I've copied your paragraph in 8b52d52. This (and other text through the docs) can be improved in 1.1.5; I had just added to an existing sentence.

**Comment by duncandewhurst** [comment]
I checked in profiles, other repos I had checked out locally and in the public templates folder on Google Drive for any examples of URLs for extension documentation which would now be broken (e.g. `http://standard.open-contracting.org/latest/en/extensions/location/`) but didn't find any.

We will have shared links like these in documents and guidance for individual publishers and possibly elsewhere (blogs etc.) that will now 404.

**Comment by jpmckinney** [comment]
I should have mentioned that I already checked all repositories for broken links :)

To avoid 404s from other sources, seen CRM issue 4401

**Comment by jpmckinney** [comment]
@duncandewhurst Now ready for review after incorporating suggested edits.

**Review by duncandewhurst — APPROVED** [review]

**Pull request merged**